src/httpd_streaming.c

```diff
@@ -617,6 +625,39 @@ int
 streaming_init(void)
 {
   int ret;
+  cfg_t *cfgsec;
+  int val;
+
+  cfgsec = cfg_getsec(cfg, "streaming");
+
+  val = cfg_getint(cfgsec, "sample_rate");
+  if (val%11025 > 0 && val%12000 > 0 && val%8000 > 0)
```

Comment:
Add spaces around operators. Also add a comment that this is a checks for libmp3lame_sample_rates. You could also consider dropping the check altogether, because you risk that it gets out of sync with what ffmpeg actually supports. And if the user configures something invalid it will fail anyway with a log message.

Reply:
I think its cleaner to provide correct values for setting up the ffmpeg/transcode stuf - if we let the values pass through ffmpeg/transcode will fail setting up and disable streaming requiring restart which people won't notice until a streaming client attaches  With this at least we have a working streaming even if its not with the params the user (thought) they'd config'd.